README.md

```diff
@@ -48,6 +48,32 @@ app.inject({
 })
 ```
 
+Different content types responses are supported by `@fastify/response-validation`, `@fastify/swagger` and `@fastify`. Please use `content` for the response otherwise Fastify itself will fail to compile the schema:
```

Comment:
```suggestion
Different content types responses are supported by `@fastify/response-validation`, `@fastify/swagger` and `fastify`. Please use `content` for the response otherwise Fastify itself will fail to compile the schema:
```